openlibrary/plugins/upstream/utils.py

```diff
@@ -557,6 +556,28 @@ def get_languages():
         _languages = sorted([web.storage(name=d.name, code=d.code, key=d.key) for d in web.ctx.site.get_many(keys)], key=lambda d: d.name.lower())
     return _languages
 
+
+@public
+def get_author_config():
+    return _get_author_config()
+
+
+@web.memoize
+def _get_author_config():
+    """Returns the author config.
+
+    The results are cached on the first invocation.
+    Any changes to /config/author page require restarting the app.
+
+    """
+    # TODO: figure out why this doesn't work
```

Comment:
Hmm :/ What's it doing?

Reply:
This was a problem when I was thinking that the `.page` file should contain the config instead of it being added via the website GUI. Now that I know that this isn't an issue.